### ISSUE TITLE
Unable to launch VS Code on Windows 7, or us CLI

### ISSUE THREAD

**Issue by justin-nodeboy** [Microsoft/vscode#55737]
Hi

So yesterday VS Code was working on my work windows 7 VDI. However this morning it just will not open.

I tried to launch it via the CLI and get the following error:

`λ Code

#
# Fatal error in c:\users\elecsa\buildagent\workspace\358518\src\v8\src\snapshot\snapshot.h, line 28
# Check failed: IsSane().
#

==== C stack trace ===============================

        v8_inspector::protocol::Debugger::API::SearchMatch::fromJSONString [0x000007FED70C145B+178411]
        V8_Fatal [0x000007FED6DA039D+125]
        v8::internal::NativesCollection<0>::GetScriptsSource [0x000007FED6F82246+9590]
        v8::Isolate::New [0x000007FED6D36EE6+326]
        GetHandleVerifier [0x00000001401AE172+1019026]
        GetHandleVerifier [0x00000001401AE04C+1018732]
        GetHandleVerifier [0x00000001401AE005+1018661]
        (No symbol) [0x000000013FE2EC40]
        (No symbol) [0x000000013FE0FE84]
        (No symbol) [0x000000013FE08DE3]
        IsSandboxedProcess [0x000000014307F4F7+46632355]
        BaseThreadInitThunk [0x0000000076AF59CD+13]
        RtlUserThreadStart [0x0000000076D5383D+29]`

Not really sure where to start with this one

**Comment by isidorn** [comment]
Can you reproduce this issue with vscode insiders https://code.visualstudio.com/insiders/

**Comment by justin-nodeboy** [comment]
@isidorn Let me try on Tuesday, I have got it working for now by doing a full uninstall followed by a re-install. I don't want to tempt fate by restarting my VDI and trying again just now as I need to make some progress today on a project.

But I'll be logging out tonight so when I'm back on Tuesday I'll give it a go. (I only installed the latest version this week from scratch but I hadn't restarted the VDI. Maybe something got stuck somewhere)

Let me try some things on Tuesday to see if it reproduces and I have less pressure to deliver something
